apps/backoffice-v2/src/pages/Document/hooks/useDocumentLogic/useDocumentLogic.tsx

```diff
@@ -43,7 +43,7 @@ export const useDocumentLogic = () => {
         );
       }
 
-      broadcastChannel.postMessage({
+      void broadcastChannel.postMessage({
```

Comment:
ESLint change.

Reply:
Stupid rule. Should we change it?

Reply:
It was a rule that was recently voted in. It was brought up to our attention that this rule catches an important class of bugs and errors.

Reply:
Just to educate myself - what does it catch? How does it help us?

Reply:
1. Human error - just recently we caught a bug due to a missing `await`
2. Try/catch behaves differently when an async return has `await` as opposed to without `await`
3. In general this rule also promotes using try/catch and catch https://typescript-eslint.io/rules/no-floating-promises/